src/engine/mod.rs

```diff
@@ -30,7 +30,7 @@ pub mod strat_engine;
 
 mod consts;
 #[allow(module_inception)]
-mod engine;
+pub mod engine;
```

Comment:
without this, the added integration test doesn't compile (it needs to import HasUuid to call pool.uuid().